float64 omega
float64 error
float64 pathX
float64 pathY
float64 robotX
float64 robotY
float64 modulDistance
float64 pathAngle
